server/bin/pbench-index.py

```diff
@@ -170,9 +181,9 @@ def main(options, name):
 
     idxctx.logger.debug("{}.{}: starting", name, idxctx.TS)
 
-    idx = Index(name, options, idxctx, INCOMING_rp)
+    idx = Index(name, options, idxctx, INCOMING_rp, ARCHIVE_rp, qdir)
```

Comment:
This isn't really a great use of `idx` either. Generally, short and abbreviated names should be used for things like "throwaway" loop variables, with longer and more meaningful names for variable of more importance and wider scope.

I suppose the fact that I'm paying attention to minor details on this review pass could be because I'm not seeing any important issues.

Again, not anything that needs to be resolved right now, but something to consider if you have to make another change for some other reason.